### ISSUE TITLE
Validation of input values

### ISSUE THREAD

**Issue by kundkingan** [mosbth/cform#24]
If a user would input harmful code there is no check for it.
At [https://github.com/mosbth/cform/blob/v2/src/HTMLForm/CForm.php#L182](url)

**Comment by mosbth** [comment]
Yes, one needs to check the value when one extracts it. But if saving it to the database one wants unescaped values, if output to a webpage, one wants escaped values. Maybee its not for CForm to decide and leave it to the user?

Maybe, I'm not sure, it coul dbe an issue when using validation and validation fails and redirects back with an unescaped value being output to the browser. This could be a thing to fix.
